addon/utils/bind-data-test-attributes.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+const TEST_SELECTOR_PREFIX = /data-test-.*/;
+
+export default function bindDataTestAttributes(component) {
+  let computedBindings = component.attributeBindings && component.attributeBindings.isDescriptor;
+  if (computedBindings) {
+    let message = `ember-test-selectors could not bind data-test-* properties on ${component} ` +
+      `automatically because attributeBindings is a computed property.`;
+
+    return Ember.warn(message, false, {
+      id: 'ember-test-selectors.computed-attribute-bindings',
+    });
```

Comment:
I'm not sure it's worth the extra complexity to be honest. I've never seen anyone use a CP for `attributeBindings` and I have quite a hard time coming up with use cases where that would be a good idea. I'd suggest keeping the warning for now, and if many users report that they are seeing this warning then we can still think about implementing it.